.github/workflows/elixir.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Elixir CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
```

Comment:
Please add a job to check for conventional commits as well. I would love to have automatic changelog based on the conventional commits. (There is preconfigured action for that: https://github.com/revolko/recipes/blob/master/.github/workflows/per-branch.yaml#L12).